lib/player.rb

```diff
@@ -0,0 +1,36 @@
+class Player
+  attr_reader :score, :inputs, :mark
+
+  def initialize (mark)
+    @mark = mark
+    @score = 0
+    @inputs = []
+  end
+
+  def save (input)
+    @inputs << input
+  end
+
+  def valid? (input)
```

Comment:
This is better as a private method since its sole responsibility is checking whether or not the player's choice is a valid one. This method isn't part of the player's public interface